packages/components/src/internal/components/ontology/OntologyTreeSearchContainer.tsx

```diff
@@ -1,25 +1,81 @@
-import React, { ChangeEvent, FC, memo, MouseEvent, useCallback, useEffect, useMemo, useState } from 'react';
+import React, { ChangeEvent, FC, memo, MouseEvent, ReactNode, useCallback, useEffect, useMemo, useState } from 'react';
 
 import { Alert, searchUsingIndex } from '../../..';
 
 import { ConceptModel, OntologyModel, PathModel } from './models';
-import { fetchAlternatePaths } from './actions';
+import { fetchAlternatePaths, getOntologyDetails } from './actions';
 
 const CONCEPT_CATEGORY = 'concept';
 const SEARCH_LIMIT = 20;
 
+interface OntologySearchInputProps {
```

Comment:
The typings of this interface could be melded with the typings of the `OntologyTreeSearchContainer` since we are allowing pass through of props aside from `ontologyModel` and `searchPathClickHandler`. I would consider changing this interface to be:

```ts
interface OntologySearchInputProps extends Omit<OntologyTreeSearchContainerProps, 'ontology' | 'searchPathClickHandler'> {
    ontologyId: string;
    searchPathClickHandler: (code: string) => void;
}
```

Reply:
TIL -- switched